SPECS/python-distro/python-distro.spec

```diff
@@ -1,19 +1,18 @@
 %define debug_package %{nil}
 Summary:        Distro - an OS platform information API
 Name:           python-distro
-Version:        1.6.0
-Release:        3%{?dist}
+Version:        1.9.0
+Release:        1%{?dist}
 License:        ASL 2.0
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Group:          Development/Languages/Python
 URL:            https://distro.readthedocs.io/en/latest/
-Source0:        https://github.com/python-distro/distro/releases/download/v%{version}/distro-%{version}.tar.gz
+Source0:        https://github.com/python-distro/distro/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
 BuildRequires:  python3-devel
-BuildRequires:  python3-setuptools
-%if %{with_check}
```

Comment:
any reason you removed the check with_check?

Reply:
I needed the packages for building as well.